### ISSUE TITLE
Rails 5 Test Approach

### ISSUE THREAD

**Issue by brymacto** [appfolio/rails-forward_compatible_controller_tests#15]
Wondering what the intended approach is for Rails 5.  I noticed that there is code in the specs to handle Rails 5/Action Pack 5 (e.g., using the dummy class), however when I run the specs using ;appraisal actionpack-5.0 rake', 0 tests are run due to the use of that dummy class instead of a minitest class.

I'm wondering if:
1. Somehow the 0 runs in Rails 5 are a result of my environment or me missing a step in running them
2. It is intentional that tests don't run in Rails 5 because the gem is intentionally a no-op in that version
3. Specs are supposed to run in Rails 5 but are broken

If 2 above is true, then can the code for running the specs in this version be removed? Or should we get it running in Rails 5 so that users can verify that the behaviour in 4 using the gem is equivalent to 5 without the gem?

Let me know your thoughts, and I am happy to do a PR related to this work.

**Comment by cshaffer** [comment]
Seems like it was 2: https://github.com/appfolio/rails-forward_compatible_controller_tests/pull/10

I have no opinion here.  Seems like it doesn't hurt to run it against Rails 5, you'd have to ask @mqchau what his motivations were.

**Comment by mqchau** [comment]
That is correct. This gem is supposed to be removed after upgrading to Rails 5. So there is not a lot of points to run tests against Rails 5.

**Comment by bboe** [comment]
I think it would be good if the tests ran in Rails 5. While they should be a no-op, when we actually updated, we learned that we needed to run in Rails 5 with this gem for a little while, thus ensuring that it doesn't break anything, even when on Rails 5, is pretty important.

If your still interested in contributing we'd love your support. Thanks!

**Comment by bboe** [comment]
Should we considered this issue closed now, or is there anything missing?
